src/scripts/sync_synonym_curation_filtering.py

```diff
@@ -150,9 +150,6 @@ def cli():
     parser.add_argument(
         '-A', '--added-outpath', required=True,
         help='Path to filtered ROBOT template TSV containing synonyms that aren\'t yet integrated into Mondo.')
-    parser.add_argument(
-        '-C', '--confirmed-outpath', required=True,
```

Comment:
Just FYI this change is erroneously showing up in this diff.

The change hadn't been pushed to the base branch when I opened this PR. But I pushed it now, yet this still shows up in the diff when it should not.

Context: I removed this param from `sync_synonym_curation_filtering.py` since `-confirmed` is not mutated by this file. It was only here as a way to save files to `reports/sync-synonym/` when they were previously saved to `tmp/`, but I changed the way that this is set up in the makefile to make more sense.